app/views/taxons/_organisations.html.erb

```diff
@@ -1,14 +1,29 @@
-<% if presented_taxon.show_organisations? %>
-  <div class="taxon-page__section-group">
+<% if presented_organisations.show_organisations? %>
+  <div class="taxon-page__section-group" data-module="toggle">
     <div class="grid-row">
       <div class="column-two-thirds">
         <h2 class="taxon-page__section-heading">Organisations</h2>
-        <%= render 'govuk_publishing_components/components/document_list',
-          items: presented_taxon.taxon_organisations,
-          margin_top: true,
-          margin_bottom: false
-      %>
       </div>
     </div>
+
+    <%= render partial: 'organisation_logos_and_list', locals: {
+      organisations_with_logos: presented_organisations.promoted_organisation_list[:promoted_with_logos],
+      organisations_without_logos: presented_organisations.promoted_organisation_list[:promoted_without_logos]
+    } %>
+
+    <% if presented_organisations.show_more_organisations? %>
+      <a href="#"
```

Comment:
I think this is a pattern of our show more toggle - the same happens for other show more links across GOV.UK, e.g https://www.gov.uk/government/publications/sri-lanka-consular-fees (the full history; hamburger menu on mobile)

Reply:
It's something toggle doesn't account for, because it's a Javascript only module that doesn't have its own styles. Even if this is happening across GOV.UK it isn't necessarily correct. See the [metadata component](https://govuk-static.herokuapp.com/component-guide/metadata/extensive_specialist_document/preview) for an example of how to implement toggle with this feature included.

Reply:
@leenagupte Here's an example in code of how to implement it: https://github.com/alphagov/govuk_publishing_components/pull/294/files#diff-71a88e94c85be5bdf32f6f31e649e8b3R30

Reply:
@andysellick Fixed! Thanks @vanitabarrett!